src/lib.rs

```diff
@@ -1,4 +1,4 @@
-//! A pure-Rust frontend for the popular PostgreSQL database.
+//! A pure-Rust frontend for the popular `PostgreSQL` database.
```

Comment:
That does not seem like a particularly good reason to put PostgreSQL in ticks :P